addons/wp/functions/fnc_taskReset.sqf

```diff
@@ -72,21 +70,37 @@ _group setVariable [QGVAR(taskAssaultMembers), nil, true];
     _x setVariable [QEGVAR(danger,disableAI), nil, true];
     _x setVariable [QEGVAR(danger,forceMove), nil, true];
 
-    // rejoin group
-    [_x] joinSilent _groupNew;
-    _x doFollow (leader _x);
+    // rejoin
+    _x doFollow _leader;
 } count _units;
 
+// exit on soft reset
+if (_softReset) exitWith {
+    _units joinSilent _group;
```

Comment:
why do we Rejoin the same old group? this i don't think is required here

Reply:
Its a pain in the ass. but sometimes getting the AI to actually follow the leader seems to require them leaving and rejoining the group. Without it, I found errors in attempting to reset taskCamp.

Reply:
ok, I see. let's hope this does not break anything